src/oasis-nexus/api.ts

```diff
@@ -556,11 +556,13 @@ export const useGetRuntimeEvents: typeof generated.useGetRuntimeEvents = (
             events: data.events.map(event => {
               const adjustedHash = event.eth_tx_hash ? `0x${event.eth_tx_hash}` : undefined
               if (
-                event.type === 'accounts.transfer' ||
-                event.type === 'accounts.mint' ||
-                event.type === 'accounts.burn' ||
-                event.type === 'consensus_accounts.deposit' ||
-                event.type === 'consensus_accounts.withdraw'
+                event.type === RuntimeEventType.accountstransfer ||
+                event.type === RuntimeEventType.accountsmint ||
+                event.type === RuntimeEventType.accountsburn ||
+                event.type === RuntimeEventType.consensus_accountsdeposit ||
+                event.type === RuntimeEventType.consensus_accountswithdraw ||
+                event.type === RuntimeEventType.consensus_accountsdelegate ||
+                event.type === RuntimeEventType.consensus_accountsundelegate_done
```

Comment:
Don't we also need `accountsundelete_start` here? (I don't know, just asking')

Reply:
These are just the ones that have 'Amount' in the event type, which `accountsundelete_start`  doesn't.